doc/source/user/building.rst

```diff
@@ -94,10 +94,23 @@ Testing
 -------
 
 Make sure to test your builds. To ensure everything stays in shape, see if
-all tests pass::
+all tests pass
+
+First install all the python dependencies that will be required for testing 
+by running the below command in the root directory::
+
+    $ pip install -r test_requirements.txt
+
+Run tests::
 
     $ python runtests.py -v -m full
 
+``-v`` flag for showing the output of tests
+
+``-m`` flag for running a module or a package as a script
+
+``full`` for complete testing (will also run tests that are marked as slow)
+
```

Comment:
It seems the flags have gotten confused here. The `-m` flag is for `runtests.py`, not `python`, and is short for the `--mode` flag, of which `full` is one of the options.

Reply:
Correcting the flags as follows

`-m full` 
`-m` flag is for `runtests.py` shorthanded notation for `--mode` flag. 
`full` complete testing mode (will also run tests that are marked as slow) 

Could you please confirm?

Reply:
Just a thought: I think these explanations belong in [testing builds](https://numpy.org/devdocs/dev/development_environment.html#testing-builds) to keep all information consolidated.